qmi/core/transport.py

```diff
@@ -435,6 +441,11 @@ def _validate_device_name(device: str) -> None:
         if not (device.upper().startswith("COM") or device.startswith("/")):
             raise QMI_TransportDescriptorException("Unknown serial port device path ({})".format(device))
 
+    @staticmethod
+    def _validate_rstcts(rtscts: bool) -> None:
+        if rtscts not in (True, False):
```

Comment:
This does accept inputs `0` and `1` as well, btw. If you want to make this strict(er), you can add also type check.